Dockerfile

```diff
@@ -22,7 +22,6 @@ RUN echo " ... adding throttling support with ZMQ and CZMQ" \
          && cd /tmp/ \
          && tar -xf /tmp/zeromq.tar.gz \
          && cd /tmp/zeromq*/ \
-         && apk add automake autoconf \
```

Comment:
this is for preparing for alpine:latest. In alpine latest automake and autoconf are missing.